elastic/kibana
# [Spaces] Added disabled features back to mappings

## Summary

We aggregate on disabledFeatures in [Spaces Usage Collector](https://github.com/elastic/kibana/blob/5e95a7679681467b763b106ee5a03d0af643c8bc/x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.ts#L38), but field was removed from mappings. Added `disabledFeatures` back to mappings.

### How to Test

1. Create a couple of spaces with disabled features.
   ```
   POST kbn:/api/spaces/space 
   {
      "name": "my-space-1",
      "id": "my-space-1",
      "description": "a description",
      "color": "#5c5959",
      "disabledFeatures": ["canvas", "discover"]
   }
   
   POST kbn:/api/spaces/space 
   {
      "name": "my-space-2",
      "id": "my-space-2",
      "description": "a description",
      "color": "#5c5959",
      "disabledFeatures": ["savedObjectsManagement", "canvas"]
   }
   ```
2. Make a request to stats endpoint and check that `disabledFeatures` counters.
    ```
    POST kbn:/internal/telemetry/clusters/_stats
    {
      "unencrypted": true, "refreshCache": true
    }
    ```

### Checklist


### For maintainers


__Fixes: https://github.com/elastic/kibana/issues/184194__

## Release note
Added `disabledFeatures` back to mappings, so it can be aggregated on.
